src/routing.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const url = require('url');
+const databaseInterface = require('./dbInterface.js');
+
+const authentificate = (login, password) => {
+  if (!login || !password) return {
+    err: '<h1>&#127819Login and password should be specified</h1>',
+    account: null
+  };
+  const account = databaseInterface.getAccount(login);
+  if (!account) return {
+    err: '<h1>&#127819Incorrect login entered :(</h1>',
+    account: null
+  };
+  if (password !== account.password) return {
+    err: '<h1>&#127819Incorrect password entered :(</h1>',
+    account: null
+  };
+  return { err: null, account };
+};
+
+const profileHandler = (req, res) => {
+  const login = url.parse(req.url, true).query.login; //const login = req.getHeader('login');
+  const password = url.parse(req.url, true).query.password; //const password = req.getHeader('password');
```

Comment:
Isn't `url.parse` deprecated?

Reply:
It's indeed deprecated, use URL constructor instead.